### PR TITLE
[cookies] switch to self hosted cookie consenst

### PR DESCRIPTION
**Pull request by stubbi**

### CODE REVIEW

File: app/views/layouts/application.html.erb [hunk 1]
```diff
@@ -135,5 +137,8 @@
     <%= link_to 'GitHub', "https://github.com/aqora-io/quantumnews" %>
   </footer>
   <span id="iab-pcm-sdk"></span><span id="iab-autofill-sdk"></span>
+  <% if Rails.application.config.cookie_consent_config_js.present? %>
```
Comment (jpopesculian) [inline app/views/layouts/application.html.erb:130]: I don't know if this script is also blocking google analytics automatically, but in the case that it is, it probably needs to be loaded before the gtag.js?

**Review by jpopesculian — COMMENTED** [review]
I trust you on this. Just double checking

File: app/views/layouts/application.html.erb (hunk 1 above)
Comment (stubbi) [reply app/views/layouts/application.html.erb:130]: @jpopesculian I can confirm it works this way for the main site. Adding `type="text/plain" data-category="marketing"` will put them in some hook to be loaded once confirmed

Comment (jpopesculian) [reply app/views/layouts/application.html.erb:130]: i mean, ideally these kind of scripts should be deferrable though. or at least loaded at the very end of the HTML document

Comment (stubbi) [reply app/views/layouts/application.html.erb:130]: You mean all of it should be put at the end of the `body`? doing it according to the doc [here](https://cookieconsent.orestbida.com/essential/getting-started.html#html)

Comment (stubbi) [reply app/views/layouts/application.html.erb:130]: ah, you mean putting the gtag thingy behind it?

Comment (stubbi) [reply app/views/layouts/application.html.erb:130]: moved it @jpopesculian

**Review by jpopesculian — APPROVED** [review]
perfect! you'll have to verify that it works. sorry, i didn't realize what they're doing with loading it as "text/plain" so that it doesn't trigger the script

**Comment by stubbi** [comment]
let's see 🚀

**Pull request merged**